org.eclipse.buildship.ui/plugin.xml

```diff
@@ -263,6 +263,50 @@
 
     <!-- properties view for tasks and projects -->
     <extension
+          point="org.eclipse.debug.ui.launchShortcuts">
+       <shortcut
+             class="org.eclipse.buildship.ui.launch.TestLaunchShortcut"
+             id="org.eclipse.buildship.ui.shortcut.test"
+             label="Gradle Test"
+             modes="run">
+          <configurationType
+                id="org.eclipse.buildship.core.launch.runconfiguration">
+          </configurationType>
+          <contextualLaunch>
+             <contextLabel
+                   label="Gradle Test"
+                   mode="run">
+             </contextLabel>
+             <enablement>
+                <with
+                      variable="selection">
+                   <iterate
+                         ifEmpty="false"
+                         operator="and">
+                      <adapt
+                            type="org.eclipse.core.resources.IResource">
+                         <test
```

Comment:
Can we define an enablement as follows
- project with Gradle nature
- selection instanceof ITextSelection